be/Cf.Application/Services/ServiceInfoServices/ServiceInfoService.cs

```diff
@@ -0,0 +1,183 @@
+﻿using Cf.Application.Services.Interfaces;
+using Cf.Contracts.Responses;
+using Cf.Domain.Aggregates.Services;
+using Cf.Domain.Enums;
+using Cf.Domain.Exceptions.Messages;
+using Cf.Domain.Exceptions;
+using Cf.Domain.Models;
+using Cf.Infrastructure;
+using Microsoft.EntityFrameworkCore;
+using Cf.Contracts.Mappers;
+
+namespace Cf.Application.Services.ServiceInfoServices
+{
+    public class ServiceInfoService : IServicelnfoService
+    {
+        private readonly Context _context;
+        private readonly IServiceWorkingDaysService _serviceWorkingDaysService;
+
+        public ServiceInfoService(Context context, IServiceWorkingDaysService serviceWorkingDaysService)
+        {
+            _context = context;
+            _serviceWorkingDaysService = serviceWorkingDaysService;
+        }
+
+        public async Task UpdateInfoAsync(string? serviceId, ServiceAdditionalInfoModel additionalInfo)
```

Comment:
In general this method is very crowded. I will provide some suggestions how to make it readable